### PR TITLE
fix tracking summary errors

### PR DESCRIPTION
**Pull request by fioriNTU**
fix for rechit and chi2 flag in Tracking summary map

### CODE REVIEW

File: DQM/TrackingMonitorClient/data/tracking_qualitytest_config_tier0.xml [hunk 1]
```diff
@@ -17,29 +17,50 @@
...
+     <TYPE>MeanWithinExpected</TYPE> 
+     <PARAM name="error">0.85</PARAM> 
+     <PARAM name="warning">0.95</PARAM> 
+     <PARAM name="mean">1.0</PARAM>
+     <PARAM name="useRMS">0</PARAM>
+     <PARAM name="useSigma">0</PARAM>
+     <PARAM name="useRange">1</PARAM>
+     <PARAM name="minEntries">0</PARAM>
+     <PARAM name="xmin">0.5</PARAM>
+     <PARAM name="xmax">1.5.0</PARAM>
```
Comment (threus) [inline DQM/TrackingMonitorClient/data/tracking_qualitytest_config_tier0.xml]: probably a typo for xmax here?

Comment (fioriNTU) [inline DQM/TrackingMonitorClient/data/tracking_qualitytest_config_tier0.xml]: Hi Tomas,
you are right, it is a typo. Anyway the test run in the expected way, I 
will close the PR, correct and open again.
Thanks for spotting!!


Il 16/09/2015 22.37, threus ha scritto:
>
> In 
> DQM/TrackingMonitorClient/data/tracking_qualitytest_config_tier0.xml 
> <https://github.com/cms-sw/cmssw/pull/11292#discussion_r39681763>:
>
> >       <PARAM name="xmax">35.0</PARAM>
> > -</QTEST>
> > -<QTEST name="MeanWithinExpected:Chi2oNDF" activate="true">
> > -     <TYPE>ContentsXRange</TYPE>
> > +</QTEST>-->
> > +<QTEST name="MeanWithinExpected:Chi2oNDF" activate="true">
> > +     <TYPE>MeanWithinExpected</TYPE>
> > +     <PARAM name="error">0.85</PARAM>
> > +     <PARAM name="warning">0.95</PARAM>
> > +     <PARAM name="mean">1.0</PARAM>
> > +     <PARAM name="useRMS">0</PARAM>
> > +     <PARAM name="useSigma">0</PARAM>
> > +     <PARAM name="useRange">1</PARAM>
> > +     <PARAM name="minEntries">0</PARAM>
> > +     <PARAM name="xmin">0.5</PARAM>
> > +     <PARAM name="xmax">1.5.0</PARAM>
>
> probably a typo for xmax here?
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/cms-sw/cmssw/pull/11292/files#r39681763>.
>



---
Questa e-mail è stata controllata per individuare virus con Avast antivirus.
https://www.avast.com/antivirus

**Comment by deguio** [comment]
please test

**Comment by deguio** [comment]
+1

**Comment by davidlange6** [comment]
+1

**Pull request merged**